recipe-client-addon/bootstrap.js

```diff
@@ -24,6 +24,7 @@ this.shutdown = function(data, reason) {
   // modules are not unloaded accidentally during tests.
   const log = LogManager.getLogger("bootstrap");
   const modules = [
+    "content/AboutPages.jsm",
```

Comment:
I figured it was a good signal that at least some of the code in this module runs in the content process and is restricted by that. Also I copied the pattern from Pocket, which puts it there too.